pages/properties/static.properties.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Static HTML",
+    "creator": "GitHub Actions",
+    "description": "Deploy static files in a repository without a build.",
+    "iconName": "html",
+    "categories": ["Pages"]
```

Comment:
We should also include language/frameworks targeted by this template in categories. It will be helpful if we use our suggestion engine for Pages in future.

Reply:
@anuragc617 I don't Pages UI is taking advantage of this today. Hence we can proceed with PR and take these separately.